ProcessesApi.Tests/DynamoDbIntegrationTests.cs

```diff
@@ -1,64 +1,83 @@
 using Amazon.DynamoDBv2;
 using Amazon.DynamoDBv2.DataModel;
-using NUnit.Framework;
+using Amazon.DynamoDBv2.Model;
 using System;
 using System.Collections.Generic;
 using System.Net.Http;
+using Xunit;
 
 namespace ProcessesApi.Tests
 {
-    public class DynamoDbIntegrationTests<TStartup> where TStartup : class
+    public class DynamoDbIntegrationTests<TStartup> : IDisposable where TStartup : class
     {
-        protected HttpClient Client { get; private set; }
-        private DynamoDbMockWebApplicationFactory<TStartup> _factory;
-        protected IDynamoDBContext DynamoDbContext => _factory?.DynamoDbContext;
-        protected List<Action> CleanupActions { get; set; }
+        public HttpClient Client { get; private set; }
+        public IDynamoDBContext DynamoDbContext => _factory?.DynamoDbContext;
+
+        public static string[] PatchByParentId { get; private set; }
+
+        private readonly DynamoDbMockWebApplicationFactory<TStartup> _factory;
 
         private readonly List<TableDef> _tables = new List<TableDef>
         {
-            new TableDef { Name = "Processes", KeyName = "id", KeyType = ScalarAttributeType.N }
+            new TableDef {
+                Name = "Processes",
+                KeyName = "id",
+                KeyType = ScalarAttributeType.S
```

Comment:
You would need to add the `RangeKeyName` and `RangeKeyType` here 
(and the MockWebApplicationFactory class)